stravalib/__init__.py

```diff
@@ -6,3 +6,6 @@
     from ._version_generated import __version__
 except ImportError:
     __version__ = "unreleased"
+
+
+__all__ = ["Client"]
```

Comment:
nice. i saw this was the fix from a flake8 maintainer for the flake8 unused import issue !

Reply:
Yes and it's also to practice to define explicitly what you are reexporting. Some tool will complain about using stuff that is not explicitly reexported

Reply:
oh - can you explain what you mean by re-exporting? i understand the import issue now.
And thanks for opening this. i know we have made a LOT of collective changes in the past week (but all good ones!!)

Reply:
It means you explicitly make it part of your api for this module. See maybe here for details https://mypy.readthedocs.io/en/stable/command_line.html?highlight=--no-explicit-reexport#cmdoption-mypy-no-implicit-reexport